src/adr/0041-james-servers-offering.md

```diff
@@ -0,0 +1,109 @@
+Its usage requires configuration to adapt it to the user use cases, wich can be 
+not obvious for people with limited knowledge/resources, and can discourage self hosting usages.
+
+Guice servers includes:
+
+ - **Cassandra - guice** (`server/container/guice/cassandra-guice`) and its LDAP counter-part
+(`server/container/guice/cassandra-guice-ldap`): about to be deprecated, this guice server relies on
+a Cassandra & ElasticSearch database without achieving distribution (as it misses a messaging component).
+
+ - **Distributed James** (`server/container/guice/cassandra-rabbitmq-guice`) and its LDAP counter part
+(`server/container/guice/cassandra-rabbitmq-ldap-guice`) enabled to achieve the distributed use case. This
+server furthermore is extensible, and ships many extra features by default. It is however hard to operate.
+
+ - **Guice memory** (`server/container/guice/memory-guice`) ships a server based on memory. Easy to boot, 
+without external dependencies, we mostly rely on it for testing purposes.
+
+ - **JPA Guice** (`server/container/guice/jpa-guice`) ships an extensible server based on JPA storage. With
+a default configuration using Derby embedded database, this server have no external dependencies by default.
+However the still large amount of configuration can make new comers confused. It currently satisfies operator 
+looking for self hosting solutions as well as those looking for an extensible server.
+
+ - **JPA SMTP** (`server/container/guice/jpa-smtp`) demonstrate a Mail Transfer Agent, dedicated to mail
+processing, that do not ship the James Mailbox, as well as Mail Delivery protocols like IMAP, JMAP or POP3.
+
+We can notice that this offer is specified in technical terms, and do not directly map to identified user needs.
+
+Also the large amount of servers makes the choice harder to make for newcomers.
+
+## Decision
+
+We should deliver a subset of mail server directly adressing users needs, and brand them according to their use cases
+and not due to their technical details.
+
+In order to get a clearer offer we should first reduce the server cardinality. This can be achieved via:
+
+ - **Spring server** deprecation and removal
+ - Merging Guice servers with their LDAP variations (see [ADR 36 about guice module choosing](0036-against-use-of-conditional-statements-in-guice-modules.md))
+ - **Cassandra-guice** deprecation and removal
+
+We should then rename our servers in order to match the use cases:
+
+ - **Basic server** addresses the self-hosting use case. It will be based on current *jpa-guice* server but with the clear intent to 
+lower adoption barriers. It exposes limited configurations in order not to confuse the user. Configuration and documentation is centered 
+around self hosting use cases.
+
+ - **Advanced server** addresses the customization use cases. It will be based on the current *jpa-guice* server. It differs from the basic server
```

Comment:
In my opinion this one is useless

Reply:
I think your comment is only relevant when taken together with the Basic Server, so your comment is about both simultaneously.